lib/logstash/outputs/influxdb.rb

```diff
@@ -29,10 +32,10 @@ class LogStash::Outputs::InfluxDB < LogStash::Outputs::Base
   config :port, :validate => :number, :default => 8086
 
   # The user who has access to the named database
-  config :user, :validate => :string, :default => nil, :required => true
+  config :user, :validate => :string, :default => nil
```

Comment:
Fair enough, but I didn't specify it. It was in the codebase already. (I just removed `:required => true` as it is not a required field for Influxdb 0.9 when authentication is disabled, which is the default I believe)